deployment/ocp/cloudigrade.yml

```diff
@@ -8,7 +8,123 @@ objects:
 - kind: DeploymentConfig
   apiVersion: v1
   metadata:
-    name: cloudigrade
+    name: ${NAME}-celery
+  spec:
+    replicas: 1
+    revisionHistoryLimit: 10
+    selector:
+      name: ${NAME}-celery
+    template:
+      metadata:
+        labels:
+          name: ${NAME}-celery
+      spec:
+        containers:
+        - name: ${NAME}-celery
```

Comment:
This container is missing readiness and liveness probes, we need some method for checking if it's healthy.